.github/workflows/rollback.yml

```diff
@@ -1,11 +1,287 @@
 name: Rollback
+run-name: Rollback the ${{ inputs.service }} service in the ${{ inputs.environment }} environment
 
 on:
   workflow_dispatch:
+    inputs:
+      service:
+        type: choice
+        description: The microservice which we want to rollback to the previous version.
+        options:
+          - inbound_mail
+          - api
+          - web
+          - webhook
+          - widget
+          - worker
+          - ws
+      environment:
+        type: choice
+        description: The environment where the service is located.
+        options:
+          - Development
```

Comment:
Let's add production first, so it's select by default. My guess is that we should almost never rollback development.